src/Pages/EditEntry.cshtml

```diff
@@ -0,0 +1,79 @@
+﻿@page
+@using ePiggyWeb.Utilities
+@model ePiggyWeb.Pages.EditEntryModel
+
+<br />
+<div class="container p-0 mt-3 mb-4">
+    <!--Displays text accordingly to the EntryType-->
+    @if (Model.EntryTypeInt == 1)
+    {
+        <h2 class="text-info">Income</h2>
+    }
+    else
+    {
+        <h2 class="text-info">Expenses</h2>
+    }
+</div>
+
+<!--Base values are taken from already existing Entry-->
+<form method="post" class="mb-5">
+    <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+    <input asp-for="EntryTypeInt"  type="hidden"/>
+    <input asp-for="Entry.Id" type="hidden"/>
+    <input asp-for="Entry.UserId" type="hidden" />
+    <input asp-for="Entry.Title" type="text"  />
+    <span asp-validation-for="Entry.Title" class="text-danger"></span>
+    <input asp-for="Entry.Amount" type="text"  />
+    <span asp-validation-for="Entry.Amount" class="text-danger"></span>
+    <input asp-for="Entry.Date" type="date" value="@Model.Entry.Date.ToShortDateString()" />
+    <select asp-for="Entry.Importance" class="ml-3">
+        <option hidden disabled selected value="@Model.Entry.Importance">@((Importance)Model.Entry.Importance)</option>
+        <option value="1">Necessary</option>
+        <option value="2">High</option>
+        <option value="3">Medium</option>
+        <option value="4">Low</option>
+        <option value="5">Unnecessary</option>
+    </select>
+    <span asp-validation-for="Entry.Importance" class="text-danger"></span>
+    <label class="text-light ml-3">Is Monthly?</label>
+    @if (@Model.Entry.Recurring)
+    {
+        <input asp-for="Entry.Recurring" type="checkbox" checked />
+    }
+    else
+    {
+        <input asp-for="Entry.Recurring" type="checkbox" />
+    }
+    <input type="submit" value="Save" class="btn btn-primary ml-5" />
+    <input asp-page-handler="Cancel" class="btn btn-danger ml-2" type="submit" value="Cancel">
+
+    <br />
+    <div class="container p-0 mt-5 mb-4">
+        <h4 class="text-info">Quotes by famous people</h4>
+    </div>
```

Comment:
I know this is totally a minor thing, but if it's easy to do, would it be possible to make the ui not move when going from Income to editing? As in, the Income label stays in exactly the same spot, but all the input fields move just a tiny bit down, would be neat if they also stayed in the same place.